packages/clerk-js/package.json

```diff
@@ -118,7 +115,7 @@
     "files": [
       {
         "path": "./dist/clerk.browser.js",
-        "maxSize": "169kB"
+        "maxSize": "175kB"
```

Comment:
❓ Is the size increase caused by React 18?

Reply:
Yes, but its a +1kb change. I just added a small buffer to bundlewatch on top of the actual increase